### PR TITLE
Test sdk coverage stage 1

### PR DESCRIPTION
**Pull request by gserranojc**
Increased coverage of parsers

### CODE REVIEW

**Comment by danielcsant** [comment]
doc generation is not working
check it with `mvn site`

File: sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      DimensionValue(
+        Dimension("minute", "eventKey", "identity", defaultDimension), 1L)),
+      timestamp, timeDimension)
+    val aggregations = Map("field" -> Some("value"))
+    val fixedDimensionsName = Seq("dim2")
+    val fixedDimensions = Some(Seq(("dim3", "value3")))
+    val fixedAggregation = Map("agg2" -> Some("2"))
+  }
+
+  "AggregateOperationsSpec" should "return a correct string" in new CommonValues {
```
Comment (arinconstrio) [inline sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala]: AggregateOperations instead  AggregateOperationsSpec

File: sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    val result = "dim1_dim2_minute DIMENSIONS: dim1|dim2|minute AGGREGATIONS: Map(field -> Some(value)) TIME: 1"
```
Comment (arinconstrio) [inline sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala]: result should be expected

File: sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  it should "return a correct keyString" in new CommonValues {
+    val result = "dim1_dim2_minute"
+    AggregateOperations.keyString(dimensionValuesT, timeDimension, fixedDimensionsName) should be(result)
+  }
+
+  it should "return a correct toKeyRow tuple" in new CommonValues {
+    val result = (Some("dim1_dim2_dim3_minute"), Row("value1", "value2", "value3", new Timestamp(1L), "2", "value"))
+    AggregateOperations.toKeyRow(
+      dimensionValuesT, aggregations, fixedAggregation, fixedDimensions, false) should be(result)
+  }
```
Comment (arinconstrio) [inline sdk/src/test/scala/com/stratio/sparkta/sdk/AggregateOperationsSpec.scala]: Pls split the execution code and the assert

**Pull request merged**